GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/MountTests.cs

```diff
@@ -210,6 +210,21 @@ public void MountRegeneratesAlternatesFileWhenMissingFromDisk()
             alternatesFilePath.ShouldBeAFile(this.fileSystem).WithContents(objectsRoot);
         }
 
+        [TestCase]
+        public void MountCanProcessSavedBackgroundQueueTasks()
+        {
+            this.Enlistment.UnmountGVFS();
+
+            string deleteFilePath = Path.Combine("Test_EPF_WorkingDirectoryTests", "1", "2", "3", "4", "ReadDeepProjectedFile.cpp");
+            string deleteDirPath = Path.Combine("Test_EPF_WorkingDirectoryTests", "1", "2", "3", "4");

```

Comment:
To more accurately simulate this scenario, you could also have the test do the following:

1. Read the file to be deleted before unmounting (to ensure it's on disk)
2. After unmounting, delete `deleteDirPath`